stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberFormatter.kt

```diff
@@ -56,8 +56,9 @@ internal sealed class PhoneNumberFormatter {
         override val countryCode = metadata.regionCode
 
         // Maximum number of digits for the subscriber number for this region.
-        private val maxSubscriberDigits = E164_MAX_DIGITS -
-            (prefix.length - 1) // prefix minus the '+'
+        private val maxSubscriberDigits = metadata.pattern.count { it == '#' }
+//        private val maxSubscriberDigits = E164_MAX_DIGITS -
+//            (prefix.length - 1) // prefix minus the '+'
```

Comment:
Can we remove these comments?

Reply:
oops - done. Also added a test tag to the phone text field for unittest